<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alert Clip Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QFrame" name="frame">
         <property name="maximumSize">
          <size>
           <width>1200</width>
           <height>80</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lblLogo">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>340</width>
              <height>200</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="resource.qrc">:/newPrefix/resources/Rider Dome Logo v2.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <property name="indent">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Maximum</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>80</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>30</pointsize>
              <weight>75</weight>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(66, 0, 198);</string>
            </property>
            <property name="text">
             <string>Video classification APP</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>135</width>
              <height>27</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnSetting">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/newPrefix/resources/setting.png</normalon>
             </iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnClose">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/newPrefix/resources/un-close.png</normalon>
             </iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <italic>true</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Triangular</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="iconSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <widget class="QWidget" name="tab">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0" columnstretch="0,0,0,0,0,0,0,0,0,0,0">
             <property name="horizontalSpacing">
              <number>5</number>
             </property>
             <item row="0" column="4">
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>28</width>
                 <height>30</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="2">
              <spacer name="horizontalSpacer_6">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>27</width>
                 <height>30</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="3" column="0">
              <widget class="QPushButton" name="btnGoToStartFront">
               <property name="font">
                <font>
                 <family>MS Shell Dlg 2</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <italic>true</italic>
                 <bold>true</bold>
                 <underline>false</underline>
                 <strikeout>false</strikeout>
                </font>
               </property>
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Go to the start position of the video&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normalon>:/newPrefix/resources/refresh.png</normalon>
                </iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QPushButton" name="btnOpenFront">
               <property name="font">
                <font>
                 <family>Consolas</family>
                 <pointsize>9</pointsize>
                 <weight>75</weight>
                 <italic>true</italic>
                 <bold>true</bold>
                 <underline>false</underline>
                 <strikeout>false</strikeout>
                </font>
               </property>
               <property name="toolTip">
                <string>Open Video</string>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normalon>:/newPrefix/resources/open.png</normalon>
                </iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="3" column="1" colspan="4">
              <widget class="QSlider" name="sliderFront">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="7">
              <spacer name="horizontalSpacer_7">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="5" column="0" colspan="5">
              <widget class="QLabel" name="lblFrontCamURL">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>25</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>Segoe UI</family>
                 <pointsize>11</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="margin">
                <number>5</number>
               </property>
              </widget>
             </item>

            <item row="0" column="2">
             <widget class="QLabel" name="AlertLabel">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>20</pointsize>
                <weight>75</weight>
                <italic>true</italic>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(66, 0, 198);</string>
              </property>
              <property name="text">
               <string>Alert Type</string>
              </property>
             </widget>
            </item>


             <item row="1" column="2">
              <widget class="QPushButton" name="btnFalseAlert">
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>50</height>
                </size>
               </property>
               <property name="text">
                <string> False Alert </string>
               </property>
              </widget>
             </item>
              <item row="1" column="1">
              <widget class="QPushButton" name="btnFaulty">
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>200</height>
                </size>
               </property>
               <property name="text">
                <string> Faulty</string>
               </property>
              </widget>
             </item>
              <item row="1" column="3">
              <widget class="QPushButton" name="btnDouble">
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>200</height>
                </size>
               </property>
               <property name="text">
                <string> Delete </string>
               </property>
              </widget>
             </item>

              <item row="1" column="0">
              <widget class="QPushButton" name="btnTrueAlert">
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>200</height>
                </size>
               </property>
               <property name="text">
                <string> True Alert </string>
               </property>
              </widget>
             </item>

              <item row="2" column="0">
              <widget class="QPushButton" name="btnCollisionAlert">
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>200</height>
                </size>
               </property>
               <property name="text">
                <string> Collision Alert </string>
               </property>
              </widget>
             </item>

              <item row="2" column="1">
              <widget class="QPushButton" name="btnDistanceAlert">
               <property name="maximumSize">
                <size>
                 <width>120</width>
                 <height>200</height>
                </size>
               </property>
               <property name="text">
                <string> Distance Alert </string>
               </property>
              </widget>
             </item>

            <item row="0" column="10">
             <widget class="QLineEdit" name="StartFrameBox">
                 <property name="placeholderText">
                     <string>Start Frame</string>
                 </property>
                 <property name="inputMask">
                     <string>999999</string>
                 </property>
             </widget>
            </item>
            <item row="0" column="11">
             <widget class="QLineEdit" name="EndFrameBox">
                 <property name="placeholderText">
                     <string>End Frame</string>
                 </property>
                 <property name="inputMask">
                     <string>999999</string>
                 </property>
             </widget>
            </item>
            <item row="1" column="11">
              <widget class="QPushButton" name="btnCrop">
               <property name="font">
                <font>
                 <family>MS Shell Dlg 2</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <italic>true</italic>
                 <bold>true</bold>
                 <underline>false</underline>
                 <strikeout>false</strikeout>
                </font>
               </property>
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Crop&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="font">
                <font>
                 <family>MS Shell Dlg 2</family>
                 <pointsize>20</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                 <underline>false</underline>
                 <strikeout>false</strikeout>
                </font>
               </property>
               <property name="text">
                <string> Crop </string>
               </property>

               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            <item row="2" column="11">
              <widget class="QPushButton" name="btnMove">
               <property name="font">
                <font>
                 <family>MS Shell Dlg 2</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <italic>true</italic>
                 <bold>true</bold>
                 <underline>false</underline>
                 <strikeout>false</strikeout>
                </font>
               </property>
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Move&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="font">
                <font>
                 <family>MS Shell Dlg 2</family>
                 <pointsize>20</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                 <underline>false</underline>
                 <strikeout>false</strikeout>
                </font>
               </property>
               <property name="text">
                <string> Move </string>
               </property>

               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>

            <item row="2" column="10">
             <widget class="QLineEdit" name="FileNameBox">
                 <property name="placeholderText">
                     <string>Filename</string>
                 </property>
             </widget>
            </item>

             <item row="0" column="1">
              <widget class="QPushButton" name="btnStartFront">
               <property name="font">
                <font>
                 <family>MS Shell Dlg 2</family>
                 <pointsize>10</pointsize>
                 <weight>75</weight>
                 <italic>true</italic>
                 <bold>true</bold>
                 <underline>false</underline>
                 <strikeout>false</strikeout>
                </font>
               </property>
               <property name="toolTip">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Start/Stop the video&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normalon>:/newPrefix/resources/play.png</normalon>
                </iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="9">
              <spacer name="horizontalSpacer_9">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>28</width>
                 <height>30</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="4" column="0" colspan="5">
              <widget class="QLabel" name="imgFrontBox">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>

               <property name="frameShape">
                <enum>QFrame::Box</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>

            </layout>
           </item>
          </layout>
         </widget>

        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resource.qrc"/>
  <include location="C:/Qt/5.15.2/msvc2019_64/bin/resource.qrc"/>
 </resources>
 <connections/>
</ui>
